iterative/resource_machine.go

```diff
@@ -134,6 +134,18 @@ func machineSchema() *map[string]*schema.Schema {
 			Optional: true,
 			Default:  "",
 		},
+		"kubernetes_node_selector_key": &schema.Schema{
+			Type:     schema.TypeString,
+			ForceNew: true,
+			Optional: true,
+			Default:  "accelerator",
+		},
+		"kubernetes_node_selector_value": &schema.Schema{
+			Type:     schema.TypeString,
+			ForceNew: true,
+			Optional: true,
+			Default:  "inferred",
+		},
```

Comment:
Would this make more sense as a `schema.TypeMap` following a pattern similar to `"metadata"`

Reply:
@0x2b3bfa0 / @tasdomas opinions?

Reply:
Yes, @dacbd, you're right: a `map` would be more appropriate.